.github/workflows/_deploy_cdk.yml

```diff
@@ -64,52 +66,57 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: "18.14"
-      # TODO we should reuse the checked out code from the previous step
-      # doing this here might introduce minutes between `build` and `deploy`, which
-      # gives opportunity for problems (someone merges between `build` and `deploy`)
+
+      # checkout the actual repo
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          path: metriport
+      # checkout the internal repo to get configs
       - name: Checkout
         uses: actions/checkout@v3
+        with:
+          repository: metriport/metriport-internal
+          # ref: ${{ inputs.deploy_env == 'production' && 'master' || 'develop' }}
+          ref: "237-one-repo-to-rule-em-all"
```

Comment:
💍

Reply:
oh, thanks for noting this, I should've reverted to `develop` - doing it now